metabase/metabase
# Cleanup icons and checkboxes in collections

Epic https://github.com/metabase/metabase/issues/23222

Changes:
- Makes icons smaller
- Removes checkbox animation when hovering over collection items

How to test:
- Open a collection with some unpinned items
- Hover over rows and make sure the checkbox is displayed immediately
- It should be possible to select items as before
- Open the archive and make sure items are selectable

<img width="1367" alt="Screenshot 2022-06-20 at 20 57 11" src="https://user-images.githubusercontent.com/8542534/174656519-d0104b08-8edb-43b2-8f14-dfb0b9ba0419.png">
